aikido_zen/errors/__init__.py

```diff
@@ -7,7 +7,7 @@
 
 def generate_default_message(kind):
     """Generates a default message based on kind of attack"""
-    return "Aikido firewall has blocked " + attack_human_name(kind)
+    return "Aikido Zen has blocked " + attack_human_name(kind)
```

Comment:
```suggestion
    return "Zen has blocked " + attack_human_name(kind)
```

Either `Zen` or `Zen by Aikido` but never `Aikido Zen` 😬